display/d.vect/lines.c

```diff
@@ -334,14 +334,20 @@ int draw_line(int type, int ltype, int line, const struct line_pnts *Points,
         /* use random or RGB column color if given, otherwise reset */
         /* centroids always use default color to stand out from underlying area
          */
-        if (custom_rgb && (ltype != GV_CENTROID)) {
-            primary_color->r = (unsigned char)red;
-            primary_color->g = (unsigned char)grn;
-            primary_color->b = (unsigned char)blu;
-            D_symbol2(Symb, x0, y0, primary_color, line_color);
+        if (Symb) {
+            if (custom_rgb && (ltype != GV_CENTROID)) {
+                primary_color->r = (unsigned char)red;
+                primary_color->g = (unsigned char)grn;
+                primary_color->b = (unsigned char)blu;
+                D_symbol2(Symb, x0, y0, primary_color, line_color);
+            }
+            else {
+                D_symbol(Symb, x0, y0, line_color, fill_color);
+            }
+        }
+        else {
+            G_warning(_("Symbol is NULL; unable to display points"));
```

Comment:
No need for this else statement (with warning). The only situation Symb is NULL is if the call ` Symb = S_read(symbol_name);` in line 325 fails (in which case a warning is issued anyway).

Reply:
Done